Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.py

```diff
@@ -683,6 +695,20 @@ def search_users(default_base_dn, page_size):
                 user_account_control = user.get('userAccountControl')[0]
                 user['userAccountControlFields'] = user_account_to_boolean_fields(user_account_control)
 
+            if user.get("msDS-User-Account-Control-Computed"):
+                user_account_control = user.get("msDS-User-Account-Control-Computed")[0]
+                user_account_to_boolean_dict = (
+                    user_account_to_boolean_fields_msDS_user_account_control_computed(
+                        user_account_control
+                    )
+                )
+                if user.get("userAccountControlFields"):
+                    user["userAccountControlFields"].update(
+                        user_account_to_boolean_dict
+                    )
+                else:
+                    user["userAccountControlFields"] = user_account_to_boolean_dict
```

Comment:
```suggestion
                user.setdefault("userAccountControlFields", {}).update(user_account_to_boolean_dict)
```

Reply:
I don't want to set default if the value does not exist.

Reply:
It's just a shorter way of writing those lines, the effect is the same.